### PR TITLE
[tests] Switch system saucelabs Android tests to Appium 1.20.2

### PR DESCRIPTION
**Pull request by ikalinin1**

### CODE REVIEW

**Review by abudevich — APPROVED** [review]

**Comment by ikalinin1** [comment]
> I think it would be 0.4.0 only

But its tests update only

**Review by uarlouski — APPROVED** [review]

File: vividus-tests/src/main/resources/properties/environment/system/saucelabs/mobile_app/ios/environment.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
-selenium.grid.appium-version=1.21.0
+selenium.grid.appium-version=1.22.0
```
Comment (valfirst) [inline vividus-tests/src/main/resources/properties/environment/system/saucelabs/mobile_app/ios/environment.properties:1]: I would move this update to a separate commit and upgraded the version for Android system tests as well

**Review by valfirst — APPROVED** [review]

**Review by uarlouski — APPROVED** [review]
